bokehjs/src/coffee/models/plots/plot_canvas.coffee

```diff
@@ -446,7 +438,7 @@ class PlotCanvasView extends Renderer.View
 
     for tool_view in tool_views
       level = tool_view.model.level
-      @levels[level][tool_view.model.id] = tool_view
+      @levels['tool'][tool_view.model.id] = tool_view
```

Comment:
Yes, historically it might have been true, but tools were never meant to be renderers. This is annotations' (specifically overlays') job to do painting of crosshair lines, selection areas, etc. I went ahead to removed  `tool` rendering level altogether to avoid confusion.